File: otani/03.php
<?php
  $str = "Now I need a drink, alcoholic of course, after the heavy lectures involving quantum mechanics.";
  $str_rep1 = str_replace(",", "", $str);
  $str_rep2 = str_replace(".", "", $str_rep1);
  $str_exp = explode(" ", $str_rep2);
  $i = 0;
  foreach($str_exp as $word){
    $res[$i++] = strlen($word);
  }
  foreach($res as $num){
    echo $num. " ";
  }
?>

File: otani/python/1/03.py
string = "Now I need a drink, alcoholic of course, after the heavy lectures involving quantum mechanics."
# 1.単語に分解
string = string.replace(",", "")
string = string.replace(".", "")
string = string.split(" ")
print(string)
# 2. 文字数にして出力
for string_len in string:
    print(len(string_len))
